corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js

```diff
@@ -286,6 +286,17 @@ hqDefine("cloudcare/js/form_entry/fullform-ui", function () {
             return self.currentIndex() !== "0" && self.currentIndex() !== "-1" && !self.atFirstIndex();
         });
 
+        self.erroredLabels = ko.computed(function () {
```

Comment:
This could use a comment on what the `erroredLabels` workflow is.

Reply:
Added it on the caller method inside `webformsession.js` instead.